src/multiplots.ts

```diff
@@ -2047,6 +2048,6 @@ export class MultiplotCom {
 
 
 const empty_container = {'name': '',
-'package_version': '0.6.1',
+'package_version': '0.6.2',
```

Comment:
Why do we set to 0.6.2 ? Is it related to next released version ? In this case, this should be 0.8.1